artemis-core/artemis/src/main/java/com/artemis/AspectSubscriptionManager.java

```diff
@@ -57,12 +79,42 @@ protected void setWorld(World world) {
      * See https://github.com/junkdog/artemis-odb/issues/551
 	 *
 	 * @param builder Aspect to match.
+	 * @param applyDefaults whether to apply default aspect
 	 * @return {@link EntitySubscription} for aspect.
 	 */
-	public EntitySubscription get(Aspect.Builder builder) {
+	EntitySubscription get(Aspect.Builder builder, boolean applyDefaults) {
+		if (applyDefaults && defaultAspect != null && builder.defaults) {
+			builder = applyDefaultAspect(builder.copy());
```

Comment:
Since we already assumed that most people don't create EntitySubscriptions in tight loops (I'd say mostly in the initialize phase), such a cache would increase the memory usage for every distinct aspect used and only helps for those aspects that are used multiple times.

Personally I think I have more unique aspects in my projects than non-unqiue aspects, so it's wasting memory for marginal improvement in startup time, if any.